CONTRIBUTING.md

```diff
@@ -40,7 +40,42 @@ approach.
 
 Please include tests in your pull request.
 
-We recommend making your pull request from a fork. See [Creating a pull
+We recommend making your pull request from a fork. See [creating a pull
 request from a
 fork](https://help.github.com/articles/creating-a-pull-request-from-a-fork/)
 for more information.
+
+## Stability
+
+It’s important that our users can depend on our product, and not to worry
+about changes in model-viewer causing regressions in their use of the
+component. However, it’s also important that we continue to improve
+model-viewer, making changes to improve ergonomics and rendering quality.
+
+In general, <model-viewer> follows semver. Major versions (`X.y.z`) are used for
+the largest features and backwards-incompatible changes (more below on that),
+minor versions (`x.Y.z`) are used for new features, and patch versions (`x.y.Z`)
+are used for bug fixes.
+
+When possible please keep your changes backwards-compatible - including
+keeping both existing APIs and rendering consistent. When that’s not possible,
+we’ll follow a three-step process across several [semver](semver.org) major
+versions (in the `0.y.z` versions leading up to 1.0, we’ll use the minor
+version - `y` - for the same purpose): 
+
+1) Add a new optional API to enable the new behavior; in the release notes,
+document the new behavior, as well that it will become the default in the
+next version (in for instance the `2.0.0` release)
+1) Make the new API default enabled (allow it to be explicitly disabled for
+the old behavior); in the release notes, document this change, and that the
+API will be removed in the next version (the `3.0.0` release)
+1) Remove the API (the `4.0.0` release)
+
+Breaking changes mostly center around anything that the developer would have
+to take action on. This includes changes in APIs, or changes to the model
```

Comment:
I think we could use a more specific definition of what we mean by public APIs, as in what is vs. isn't covered by this deprecation plan. We should also mention Three.js updates, which are breaking in a sense, but I believe also do not follow this process.

Reply:
Would "documented API" rather than "public API" be clearer?

Ideally three.js would follow the same process, acknowledging that they're not committing to the same sort of thing so we're doing our best to absorb their changes into our process.

When they make a small rendering fix, that falls into the small tweaks category that doesn't require cross-version documentation.

When we've made bigger rendering changes we've done our best to communicate them ahead of time or at the latest in the release notes. I think that's a miss here - changes that aren't part of the documented API, but are significant, will be added to the release notes under the "breaking changes" heading.

Let me know how that sounds.

Reply:
Would love to have @cdata weigh in here. I believe the three.js thing had more to do with transitive dependencies. +1 for "documented API".